doc/sphinx/addendum/implicit-coercions.rst

```diff
@@ -125,38 +125,44 @@ term consists of the successive application of its coercions.
 Declaring Coercions
 -------------------------
 
-.. cmd:: Coercion @qualid : @class >-> @class
+.. cmd:: Coercion @reference : @class >-> @class
+         Coercion @ident {? @univ_decl } @def_body
 
-  Declares the construction denoted by :token:`qualid` as a coercion between
-  the two given classes.
+   :name: Coercion; _
 
-  .. exn:: @qualid not declared.
+  The first form declares the construction denoted by :token:`reference` as a coercion between
+  the two given classes.  The second form defines :token:`ident`
+  just like :cmd:`Definition` :n:`@ident {? @univ_decl } @def_body`
+  and then declares :token:`ident` as a coercion between it source and its target.
+  Both forms support the :attr:`local` attribute, which makes the coercion local to the current section.
+
+  .. exn:: @ident not declared.
      :undocumented:
 
-  .. exn:: @qualid is already a coercion.
+  .. exn:: @ident is already a coercion.
```

Comment:
This change is not justified (and it is probably the same for the other error messages). If you declare a coercion using the first form, then the error message can talk about a previously declared constant and therefore it may need to print a qualified identifier and not just a simple identifier.